### PR TITLE
LRQA-72141 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-78339
https://issues.liferay.com/browse/LRQA-78342

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
Hey @melodywu, please review and make sure all set up are in the Given block.

**Comment by melodywu** [comment]
ci:test:sf

File: modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase [hunk 1]
```diff
@@ -527,37 +539,37 @@ definition {
...
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfn userln");
 
-		Navigator.gotoPage(pageName = "2B Account Page");
+			Navigator.gotoPage(pageName = "2B Account Page");
 
-		task ("Sign in as Super User and add additional user into an account") {
 			User.logoutAndLoginPG(
 				userLoginEmailAddress = "test@liferay.com",
 				userLoginFullName = "Test Test");
```
Comment (patriciajeanperez) [inline modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase:575]: @melodywu Please review the steps here

**Review by patriciajeanperez — CHANGES_REQUESTED** [review]
@melodywu Nice work, Melody! Please review all the tests again to make sure that we are removing unnecessary steps. 

These steps may be added before this change, but when we are refactoring, we want to take this chance to take a look at the entire test and see how we can improve them. In this case, when we apply BDD to the test, it can help us highlight which steps we can remove to simplify the test.

I saw a lot of log out and log in steps that these tests do not really need. Please review them one by one before resending. Thank you.

cc @sharryxue

**Comment by patriciajeanperez** [comment]
@melodywu Please also write all the tickets in the title and description of the PR. It looks like this PR is working on two subtasks but only one is written here.

File: modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase [hunk 2]
```diff
@@ -43,57 +43,53 @@ definition {
...
+				userLoginFullName = "userfn userln");
 
-		Navigator.gotoPage(pageName = "2B Account Page");
+			Navigator.gotoPage(pageName = "2B Account Page");
+		}
 
-		Accounts.addBusinessAccount(
-			accountDescription = "Accounts Name Description for Test",
-			accountName = "Accounts Name for Test");
+		task ("adds account via widget as Account Admin") {
```
[the remark this answers is not in the document]
Comment (melodywu) [reply modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase:82]: Done and thanks

File: modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase [hunk 3]
```diff
@@ -358,53 +354,59 @@ definition {
...
 
-		User.logoutAndLoginPG(
-			userLoginEmailAddress = "userea@liferay.com",
-			userLoginFullName = "userfn userln");
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfn userln");
 
-		Navigator.gotoPage(pageName = "2B Account Page");
+			Navigator.gotoPage(pageName = "2B Account Page");
```
[the remark this answers is not in the document]
Comment (melodywu) [reply modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase:386]: Done and thank

File: modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase [hunk 4]
```diff
@@ -413,51 +415,61 @@ definition {
...
```
[the remark this answers is not in the document]
Comment (melodywu) [reply modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase:447]: Done

[the remark this answers is not in the document]
Comment (melodywu) [reply modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase:447]: Done

File: modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase (hunk 1 above)
Comment (melodywu) [reply modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase:575]: done

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test

**Comment by melodywu** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**